frontend/src/components/dashboard/aliases/BlockLevelSlider.module.scss

```diff
@@ -52,26 +52,59 @@ $trackLineHeight: 4px;
       --trackLineOffset: calc(
         #{$iconHeight} + (var(--thumbDiameter) / 2) - (#{$trackLineHeight} / 2)
       );
-      position: relative;
       width: 100%;
-      cursor: pointer;
+      position: relative;
       padding-top: $iconHeight;
-      // The thumb, its labels, and the track are absolutely positioned and thus
-      // don't take up space by default. Hence, we need to add bottom padding
-      // for them so no other content overlaps them:
-      // Note: when modifying this value, make sure to also modify it
-      // in the .isFree section down below:
-      padding-bottom: calc(
-        var(--thumbDiameter) + #{$trackLineHeight} + #{$stopLabelHeight}
-      );
-
-      .track-line {
-        position: absolute;
-        background-color: $color-light-gray-20;
-        height: $trackLineHeight;
-        top: var(--trackLineOffset);
 
+      .track {
         width: 100%;
+        cursor: pointer;
+        // The thumb, its labels, and the track are absolutely positioned and thus
+        // don't take up space by default. Hence, we need to add bottom padding
+        // for them so no other content overlaps them:
+        // Note: when modifying this value, make sure to also modify it
+        // in the .isFree section down below:
+        padding-bottom: calc(
+          var(--thumbDiameter) + #{$trackLineHeight} + #{$stopLabelHeight}
+        );
+
+        .track-line {
+          position: absolute;
+          background-color: $color-light-gray-20;
+          height: $trackLineHeight;
+          top: var(--trackLineOffset);
+          width: 100%;
+        }
+
+        .thumb-container {
+          position: absolute;
+          transform: translateX(-50%);
+          cursor: pointer;
+          // The thumb has a z-index of 1 to overlap .track-stop-promotional,
+          // which is also absolutely positioned but occurs later in the DOM,
+          // and therefore would overlap the thumb.
+          z-index: 1;
```

Comment:
This is new, and needed because `.track-stop-promotional` now occurs later than `.thumb-container` (because I had to move it out of the slider track).